gffutils/interface.py

```diff
@@ -1267,6 +1267,128 @@ def child_gen():
             ):
                 yield intron
 
+    def create_splice_sites(
+        self,
+        exon_featuretype="exon",
+        grandparent_featuretype="gene",
+        parent_featuretype=None,
+        merge_attributes=True,
+        numeric_sort=False,
+    ):
+        """
+        Create introns from existing annotations.
+
```

Comment:
Please update the docstring to refer to splice sites everywhere instead of introns.